kld/src/cli/main.rs

```diff
@@ -177,6 +178,12 @@ enum Command {
         #[arg(long)]
         target: String,
     },
+    /// LSP method - List Protocols for a node
+    LspListProtocols {
+        /// Node ID to query, if not provide then query on the local node
```

Comment:
Just a minor grammar suggestion.
```suggestion
        /// Node ID to query. If not provided then query the local node
```